### PR TITLE
Test omp display env

### PR DESCRIPTION
**Pull request by mjcarr458**
Unsure how to properly test the print out of the function.

### CODE REVIEW

File: tests/5.1/display_env/test_omp_display_env.c [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#include <math.h>
+
+int errors;
+
+int test_omp_display_env(){
+	check = omp_display_env(1);
+}
+
+int main(){
+	test_omp_display_env();
```
Comment (tob2) [inline tests/5.1/display_env/test_omp_display_env.c:24]: Shouldn't this use:
```suggestion
	OMPVV_REPORT_AND_RETURN (test_omp_display_env());
```

File: tests/5.1/default/test_task_target_default_firstprivate.c [hunk 2]
```diff
@@ -0,0 +1,64 @@
+//------------test_task_target_default_firstprivate.c------------
```
Comment (tmh97) [inline tests/5.1/default/test_task_target_default_firstprivate.c:1]: I think this test is for your previous PR and shouldn't be included here? If so just go ahead and delete this test 👍

**Comment by mjcarr458** [comment]
wrap in target region call report and return

File: tests/5.1/display_env/test_omp_display_env.c [hunk 3]
```diff
@@ -0,0 +1,27 @@
+//---------test_omp_display_env.c-------------------
```
Comment (tmh97) [inline tests/5.1/display_env/test_omp_display_env.c:1]: change directory to 'runtime_calls'

**Review by tmh97 — APPROVED** [review]

**Comment by mjcarr458** [comment]
Making changes to run_test.sh in #482 to support this test.

File: tests/5.1/runtime_calls/test_omp_display_env.c [hunk 4]
```diff
@@ -0,0 +1,27 @@
+//---------test_omp_display_env.c-------------------
+//
+//OpenMP API Version 5.1 Aug 2021
+//
+//Tests the behavior of omp_display_env. This test 
+//should print out the OpenMP version number and 
+//the initial values of the ICV's associated with 
+//the environment variables.
+//------------------------------------------------
```
Comment (nolanbaker31) [inline tests/5.1/runtime_calls/test_omp_display_env.c:9]: lgtm, i would adjust the file header a bit.
i would add some equal signs before and after the dashes on lines 1 & 9, add a space after each `//`, and and an empty line between lines 8 & 9. (https://crpl.cis.udel.edu/ompvvsollve/documentation/codingstandards/)

**Review by nolanbaker31 — APPROVED** [review]

**Pull request merged**